cores/arduino/stm32/analog.h

```diff
@@ -49,14 +49,20 @@ extern "C" {
 #endif
 
 /* Exported functions ------------------------------------------------------- */
+#if defined(HAL_ADC_MODULE_ENABLED) && !defined(HAL_ADC_MODULE_ONLY)
+uint32_t get_adc_channel(PinName pin, uint32_t *bank);
+uint32_t get_adc_internal_channel(PinName pin);
+#endif
+#if defined(HAL_DAC_MODULE_ENABLED) && !defined(HAL_DAC_MODULE_ONLY)
+uint32_t get_dac_channel(PinName pin);
+#endif
 void dac_write_value(PinName pin, uint32_t value, uint8_t do_init);
 void dac_stop(PinName pin);
```

Comment:
dac_write_value() and dac_stop() should also be under compilation switch HAL_ADC_MODULE_ENABLED ...

Reply:
Right. Was not an issue and not linked to this PR anyway best to do this 😉. Done